gtda/time_series/embedding.py

```diff
@@ -282,7 +282,7 @@ def resample(self, y, X=None):
 class SingleTakensEmbedding(BaseEstimator, TransformerResamplerMixin):
     """Representation of a single univariate time series as a point cloud.
 
-    Based on a time-delay embedding technique named after F. Takens.[1]_ Given
+    Based on a time-delay embedding technique named after F. Takens. [1]_ Given
```

Comment:
Indeed, it is even more correct to have the full stop after the citation IMO. Sorry for the mistake.

Reply:
Other instances of this badness are in:
- `make_mapper_pipeline`
- `plot_static_mapper_graph` and `plot_static_mapper_graph` (`layout` parameter docs in both cases)
- `SingleTakensEmbedding` (top-level docs, fourth paragraph)